replica/replica.go

```diff
@@ -182,7 +182,10 @@ func construct(readonly bool, size, sectorSize int64, dir, head string, backingF
 	r.info.Parent = r.diskData[r.info.Head].Parent
 
 	r.insertBackingFile()
-
+	if err := preload(&r.volume); err != nil {
```

Comment:
This error is too generic and has a side effort of preloading the volume. We can think if it's proper to preload the volume, but if we only need to deal with no file extent map support, we should try to solve that issue only.

Reply:
@yasker, i was thinking of other approach also, which is to create a temporary file and send ioctl (FIEMAP and FIBMAP) and if returns no error we can go ahead and start replica else we can error out. i'm thinking to add this code in `server.create()` function.
Shall i go with the above approach ?

Reply:
roger that.